client/src/ui/molecules/login/index.scss

```diff
@@ -4,8 +4,14 @@
 .auth-container {
   margin-top: $base-unit * 2;
 
-  @media #{$mq-large-desktop-and-up} {
-    margin-top: 0;
+  /* This media condition could be removed as margin-top isn't changing from above */
+  // @media #{$mq-large-desktop-and-up} {
+  //   margin-top: 0;
+  // }
```

Comment:
Is this debugging?

Reply:
This was just a comment I left as a note to whoever reviewed before completely removing it. I will remove lines 7-10 in my next push.